### DISCUSSION TITLE
Add an option to show the first decimal of your pp if your play has less than 100

### DISCUSSION THREAD

**Discussion by ziv-vy** [ppy/osu#30561] [Ideas] [answered]
Would be cool to know your exact pp.
![image](https://github.com/user-attachments/assets/43d0474e-5e48-4720-be93-faa137fe39d2)

**Comment by peppy** [comment]
We don't show decimal pp anywhere. I'm not sure it's that useful to have.

**Reply by Joehuu** [reply]
In the website, it's shown as a tooltip.

<img width="126" alt="Screenshot 2024-11-13 at 9 57 52 PM" src="https://github.com/user-attachments/assets/a3fa102c-10be-47e8-9750-1de7eeae8cbe">

**Reply by peppy** [reply, marked as answer]
that's quite silly
